src/app/appmarket/appinstance/app-log-access/app-log-access.component.ts

```diff
@@ -27,15 +29,17 @@ export class AppLogAccessComponent implements OnInit {
                 podInfos => {
                     this.podInfos = podInfos;
                     this.selectedPodInfo = podInfos[0]
-                    this.logService.getLogsFromPod(this.appInstanceId, this.selectedPodInfo.name)
+                    this.containers = this.selectedPodInfo.containers
+                    this.selectedContainer = this.containers[0]
```

Comment:
make sure that there is at least 1 string in this.containers to avoid problems with empty / null objects

Reply:
@pgiertych will you add some check and maybe log in the console for this?